sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/Postgresql12DatabaseIT.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.ts.reactive;
+
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+
+import io.quarkus.test.bootstrap.PostgresqlService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+@EnabledIfSystemProperty(named = "ts.redhat.registry.enabled", matches = "true")
+public class Postgresql12DatabaseIT extends AbstractReactiveDatabaseIT {
```

Comment:
This test will never be run in the bare metal cause of this property + there is no respective OpenshiftTest for Postgresql12.
Please add the OpenShift test for it

Reply:
Postgresql test coverage changed.